src/components/reusuables/cards/Testimonialcard.js

```diff
@@ -0,0 +1,20 @@
+import React from "react"
+import { StaticImage } from "gatsby-plugin-image"
+
+const Testimonialcard = (props) => {
```

Comment:
Destructure props like the last review.
(props) should be ({heading, subtitle, etc})